tcs/base/fn_onPlayerRespawn.sqf

```diff
@@ -9,6 +9,14 @@ if (TCS_var_saveAndRestoreLoadouts) then {
 	player setUnitLoadout TCS_var_playerLoadout;
 };
 
+/** 
+	In my testing, this specific case ended up with the player missing framework radios on respawn,
+	thus I add this line of code to add them back.
+*/
+if ((TCS_var_addRadiosToPlayers) and (not TCS_var_saveAndRestoreLoadouts)) then {
+	[] call TCS_fnc_addConfigurationRadiosToPlayer;
+};
+
```

Comment:
I'm not sure about this behavior, what do we expect of the MM in this case? He doesn't want the loadouts saved but wants the radios on respawn? Does that make sense?

Reply:
This could be useful for test map Zeus missions. We can discuss it more in #mm-scripts

Reply:
Consider: The MM wants a mission where your loadouts are not reset on death, so when you respawn you have whatever you died with. Thus they set saveAndRestore false. Then they use framework to give radios. The natural expected behaviour is that if you die with a (say) 343, then you respawn with the 343. I tested this case and I lost all my framework radios on respawn, so I added these lines to add them back.